echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiator.java

```diff
@@ -48,29 +50,42 @@ public void call(Pipeline pipeline) {
       log.warn("Triggering {} due to {}", pipeline, pipeline.getTrigger());
       counter.increment("orca.requests");
 
-      String runAsUser = null;
-      if (pipeline.getTrigger() != null) {
-        runAsUser = pipeline.getTrigger().getRunAsUser();
-      }
-
-      Observable<OrcaService.TriggerResponse> response;
-      if (fiatEnabled && runAsUser != null && !runAsUser.isEmpty()) {
-        response = orca.trigger(pipeline, pipeline.getTrigger().getRunAsUser());
-      } else {
-        response = orca.trigger(pipeline);
-      }
-      response.subscribe(this::onOrcaResponse, this::onOrcaError);
+      createTriggerObservable(pipeline)
+        .retry((count, error) -> {
+          if (count <= 2 && isRetryable(error)) {
+            log.error("Retrying pipeline trigger, attempt {}/3: {}", count + 1, pipeline);
```

Comment:
Planning on making the number of retries configurable?

Reply:
Yes good thought.

Reply:
LGTM otherwise, not sure if you're gating on me or waiting for ajordens :D

Reply:
Drowning in PR emails so I missed this one.

Reply:
General comment about the variety of ways we're doing retrys throughout Spinnaker, some with configurable retries, some with exponential back-off, some in loops, etc.

Specific to this, maybe some backoff and a higher default retry count. It'd probably be fine if we retried for upwards of 30s?